panic: interface conversion: interface {} is nil, not map[string]interface {}

Hey :wave:

I tried using your extension today, but unfortunately am getting this error:

```
gh pairing-with gr2m
panic: interface conversion: interface {} is nil, not map[string]interface {}

goroutine 1 [running]:
main.cli()
        github.com/schustafa/gh-pairing-with/main.go:105 +0x5b8
main.main()
        github.com/schustafa/gh-pairing-with/main.go:48 +0x7c
```

Let me know if there is anyway I can help to debug

--- a/main.go
+++ b/main.go
@@ -102,7 +102,11 @@ func cli() error {
 		return fmt.Errorf("could not unmarshal: %w", err)
 	}
 
-	data := graphqlResponse["data"].(map[string]interface{})
+	data, ok := graphqlResponse["data"].(map[string]interface{})
+
+	if !ok {
+		return fmt.Errorf("could not parse response.\n\nyou may need to add the appropriate scopes to your token.\ntry running the following:\n\tgh auth refresh --scopes user:email,read:user")
+	}
 
 	for _, user := range data {
 		if user == nil {

